tests/testthat/test-all.R

```diff
@@ -125,7 +125,7 @@ test_that("read Socrata JSON as default", {
   expect_equal(1007, nrow(df), label="rows")
   expect_equal(11, ncol(df), label="columns")
   expect_equal(c("character", "character", "character", "character", "character", 
-                 "character", "character", "character", "character", "character", 
+                 "character", "logical", "character", "character", "character", 
```

Comment:
Yes, it is for `needs_recoding`. It had been coming in as a "character" but with this code change it started coming in as a "logical". I think using `fromJSON()` is what caused this change.